### PR TITLE
feat: CL Anoncreds Crypto services

### PR DESCRIPTION
**Pull request by kgoncharov**
**Title:**
CL Anoncreds Crypto services

**Description:**
* Fourth PR to support CL in `af-go` [#180](https://github.com/hyperledger/aries-framework-go/issues/180)
* This PR adds high-level Crypto services to work with CL Anoncreds with help of `ursa` lib

**Summary:**
* added high-level CL crypto Issuer API
* added high-level CL crypto Prover API
* added high-level CL crypto Verifier API
* added ursa-implementations for the services
* added unit tests for issue credential flow with CL crypto
* added unit tests for present proof flow with CL crypto

### CODE REVIEW

**Review by ashcherbakov — APPROVED** [review]

**Review by Baha-sk — APPROVED** [review]
lgtm

File: pkg/cl/api.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package cl
```
Comment (troyronda) [inline pkg/cl/api.go:7]: The `cl` package should not be at the root of the `pkg` hierarchy.

Comment (troyronda) [reply pkg/cl/api.go:7]: Would this `cl` package be similar to `jwt`, `verifiable`, ... under the pkg/doc hierarchy?

(e.g., `pkg/doc/cl`)?

Comment (kgoncharov) [reply pkg/cl/api.go:7]: Almost, it's more like high-level CL crypto services, not strongly attached to credentials/docs. But I guess it's ok to move them to doc. 
btw, our next step is to implement indy credential attachments which will definitely go to `pkg/doc`

Comment (kgoncharov) [reply pkg/cl/api.go]: @troyronda done

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**